lti_consumer/api.py

```diff
@@ -180,29 +190,65 @@ def get_lti_1p3_content_url(config_id=None, block=None, hint=""):
         b. Deep Linking is configured, but a single ltiResourceLink was selected.
     2. The Deep Linking content presentation URL if there's more than one
        Lti DL content in the database.
+
+    The launch_information parameter is a dictionary that contains data necessary and related to an LTI 1.3 launch.
+    It provides a mechanism for the context to provide contextually defined or stored launch information to the library.
+    The schema of launch_information may change as we support additional claims or LTI specifications.
+
+    Below are the currently support keys for launch_information.
+
+    config_id (required; string): the config_id field of an LtiConfiguration to use for the launch
```

Comment:
in most of the calls you leave config id / block id in place which makes this one redundant. From messing around on 1645 I think we probably should leave it out of the launch info and replace the config/block optional pair with just passing in a config directly.

So most calls look like lti_whatever_thing(lti_config, launch_info)

Currently everything would get the config by just doing get_config(optional id, optional block). But we can break those up later so that we can get different config satisfying objects from different paths, some bound to blocks, some not, some able to load their own blocks.

Reply:
from discussion in slack the config ID in the launch data is probably the one to keep, I'll look into removing the other one while trying to remove the block argument in my own refactor